.github/workflows/ci.yml

```diff
@@ -10,13 +10,19 @@ on:
       - master
 jobs:
   ci:
-    name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
+    name: Checks and tests on OTP-${{matrix.otp_vsn}}, ${{matrix.os}}, ${{matrix.profile}}
     runs-on: ${{matrix.os}}
 
     strategy:
       matrix:
-        otp_vsn: [20.3, 21.3, 22.3, 23.1]
+        otp_vsn: [20.3, 21.3, 22.3, 23.3, 24.0]
         os: [ubuntu-latest]
+        profile: [test, ranch20]
+        exclude:
+            - otp_vsn: 24.0     # ranch 1.x does not work on OTP24+
```

Comment:
We are using ranch 1.8 with OTP 24.0.5.
I suspect any problems with 1.x and OTP 24 were resolved in 1.8

Reply:
Ah, quite possible 1.8 might work. We just don't have rebar.config profile that enforces 1.8. Do you suggest I should add `ranch18` profile and add it to the matrix as well? Or it's fine as it is now?